test/gpu_reference_kernel.cpp

```diff
@@ -447,6 +482,13 @@ struct gpu_reference_conv_2d : gpu_reference_kernel_base
                                  wei.data.data(),
                                  sizeof(TRef) * wei_sz,
                                  hipMemcpyHostToDevice) == hipSuccess);
+                // TODO(Amber): copy output before computation because output may
```

Comment:
Usual advice about doxygen formatting and github username. Also I do not recommend copying comments. But you can use the `\anchor` tag like
```C++
/// \anchor test_naive_gpu_kernel_why_copy_output_before_computation
/// \todo ...
/// ...
```
and then simply
```C++
/// \ref test_naive_gpu_kernel_why_copy_output_before_computation
```

Reply:
[resolved]